export.config.json

```diff
@@ -16,7 +16,8 @@
     "Checkbox": "checkbox",
     "CheckboxGroup": "checkboxgroup",
     "Radio": "radio",
-    "Switch": "switch",
+    "Switch": "Switch",
+    "SearchBar": "searchBar",
```

Comment:
If I'm not mistaken, node module name can't contain Uppercase letter. Please use lowercase instead